invenio_oauthclient/contrib/cern.py

```diff
@@ -190,15 +194,38 @@ def account_info(remote, resp):
     """Retrieve remote account information used to find local user."""
     # Query CERN Resources to get user info and groups
     response = remote.get(REMOTE_APP_RESOURCE_API_URL)
+    current_app.logger.error(response.data)
     res = get_dict_from_response(response)
 
     email = res['EmailAddress'][0]
-    common_name = res['CommonName'][0]
+    external_id = res['PersonID'][0]
+    # nice = res['CommonName'][0]
+    # name = res['DisplayName'][0]
 
     # FIXME get user informations accordly with invenio-userprofiles
-    return dict(email=email.lower(), nickname=common_name)
+    return dict(email=email.lower(),
```

Comment:
If `invenio-userprofiles` is installed you can add `profile=dict(nickname=nice, ...)`